libraries/AP_DroneCAN/AP_DroneCAN.cpp

```diff
@@ -346,56 +343,12 @@ void AP_DroneCAN::loop(void)
 
         canard_iface.process(1);
 
-        if (_SRV_armed) {
-            bool sent_servos = false;
-
-            if (_servo_bm > 0) {
-                // if we have any Servos in bitmask
-                uint32_t now = AP_HAL::native_micros();
-                const uint32_t servo_period_us = 1000000UL / unsigned(_servo_rate_hz.get());
-                if (now - _SRV_last_send_us >= servo_period_us) {
-                    _SRV_last_send_us = now;
-                    if (option_is_set(Options::USE_HIMARK_SERVO)) {
-                        SRV_send_himark();
-                    } else {
-                        SRV_send_actuator();
-                    }
-                    sent_servos = true;
-                    for (uint8_t i = 0; i < DRONECAN_SRV_NUMBER; i++) {
-                        _SRV_conf[i].servo_pending = false;
-                    }
-                }
-            }
-
-            // if we have any ESC's in bitmask
-            if (_esc_bm > 0 && !sent_servos) {
-#if AP_DRONECAN_HOBBYWING_ESC_SUPPORT
-                if (option_is_set(Options::USE_HOBBYWING_ESC)) {
-                    SRV_send_esc_hobbywing();
-                } else
-#endif
-                {
-                    SRV_send_esc();
-                }
-            }
-
-            for (uint8_t i = 0; i < DRONECAN_SRV_NUMBER; i++) {
-                _SRV_conf[i].esc_pending = false;
-            }
-        }
-
-        led_out_send();
-        buzzer_send();
-        rtcm_stream_send();
         safety_state_send();
```

Comment:
safety and arm state need to be high priority as ESCs may disarm if they don't get that state regularly